backend/src/main/java/org/cryptomator/hub/api/VaultResource.java

```diff
@@ -362,13 +365,14 @@ public Response createOrUpdate(@PathParam("vaultId") UUID vaultId, @Valid @NotNu
 		//update new or existing vault
 		vault.name = vaultDto.name;
 		vault.description = vaultDto.description;
-		vault.archived = vaultDto.archived;
+		vault.archived = isCreated ? false : vaultDto.archived;
```

Comment:
Although understandable from user perspective, this actually changes the logic and must be noted in the `@Operation( ... description="")` field and we should also cover this in a unit test.

@overheadhunter From the backend view, it does not matter if we create directly an archived vault. What do you think?

Reply:
Another reason is that I didn't want to include an `archived` column in `CreateVaultEvent`. So I came to the conclusion that it wouldn't be that useful to create an archived vault.

Reply:
Fixed in 8b825b4fd9a1c4fae9c859d447b0218af055bb1e.